.github/workflows/container-version.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  schedule:
+    - cron: '37 13 * * *'
```

Comment:
Why at 13:37 daily?

Reply:
It's just a daily check. 1337 is just a random time :p 
We should see how it works/runs and adjust cron to our likings.

Reply:
Yeah, random :wink: